aws/src/integration/java/org/apache/iceberg/aws/s3/S3TestUtil.java

```diff
@@ -29,4 +35,18 @@ public static String getBucketFromUri(String s3Uri) {
   public static String getKeyFromUri(String s3Uri) {
     return new S3URI(s3Uri).key();
   }
+
+  /**
+   * Skip a test if the Analytics Accelerator Library for Amazon S3 is enabled.
+   *
+   * @param properties properties to probe
+   */
+  public static void skipIfAnalyticsAcceleratorEnabled(
+      S3FileIOProperties properties, String message) {
+    boolean isAcceleratorEnabled = properties.isS3AnalyticsAcceleratorEnabled();
+    if (isAcceleratorEnabled) {
+      LOG.warn(message);
+    }
+    Assumptions.assumeThat(!isAcceleratorEnabled).describedAs(message).isTrue();
```

Comment:
this should be a static import. I'm surprised that checkstyle didn't complain here about this

Reply:
Right, updated it!